mmdet/apis/ote/apis/detection/ote_utils.py

```diff
@@ -102,6 +103,21 @@ def on_train_batch_end(self, batch, logs=None):
         super().on_train_batch_end(batch, logs)
         self.update_progress_callback(self.get_progress())
 
+    def on_epoch_end(self, epoch, logs=None):
+        self.past_epoch_duration.append(time.time() - self.start_epoch_time)
+        self.__calculate_average_epoch()
+        score = None
+        if hasattr(self.update_progress_callback, 'metric') and isinstance(logs, dict):
+            score = logs.get(self.update_progress_callback.metric, None)
+        self.update_progress_callback(self.get_progress(), score)
+
+    def __calculate_average_epoch(self):
```

Comment:
I'd suggest to rename this method to something like `__calculate_average_epoch_duration` or `__update_average_epoch_duration`.

Reply:
This `__calculate_average_epoch` method is copied from its parent class `ote_sdk.usecases.reporting.time_monitor_callback.TimeMonitorCallback`. Because it is declared as a private method, it's not allowed to call it from here. If it is changed to a protected method in the parent class, I'll remove `__calculate_average_epoch` here.

Reply:
I see, let's leave it as is then.